### ISSUE TITLE
probably doesnt work

### ISSUE THREAD

**Issue by hacknorris-code** [Donnnno/Arcticons-Linux#7]
im talking about contents (or its lacking) in icon.theme file ...

**Comment by Donnnno** [comment]
it was an experiment, but we stopped working on it for the time being

**Comment by hacknorris-aka-penguin** [comment]
burh no... its good :/

**Comment by Donnnno** [comment]
This looks really promising tho!

https://fosstodon.org/web/@iooioio/108023516654561996

**Comment by hacknorris-aka-penguin** [comment]
means? there is nothing except of link here..

**Comment by Donnnno** [comment]
it's a link to a post, someone is working on a desktop version:

![image](https://user-images.githubusercontent.com/31142286/160259223-c419e600-fa67-48d6-be9f-dc86945b5978.png)

**Comment by hacknorris-aka-penguin** [comment]
saw but there is no source...

**Comment by jtrees** [comment]
Source is here: https://github.com/jtrees/Arcticons#desktop-icon-theme

You have to run the script to generate the `.theme` file, like so:

```sh
cd freedesktop-theme
./generate.sh
```

It's still really bare bones, since I merely added icons for the apps I use. Feel free to open an issue on the fork if you're missing something specific.

**Comment by hacknorris-aka-penguin** [comment]
running this doesnt create file...
![Przechwycenie obrazu ekranu_2022-03-28_19-48-32](https://user-images.githubusercontent.com/97974194/160457020-e0da0af0-ea86-487f-acf6-4ecc724b3553.png)

**Comment by jtrees** [comment]
Oh oops, you're right. I just pushed a fix. Try again (`freedesktop` branch).

**Comment by hacknorris-aka-penguin** [comment]
l8er im sleepin

**Comment by Donnnno** [comment]
I wanted to install it, but Steam Deck's KDE version can't read the theme file 🙃

**Comment by jtrees** [comment]
> I wanted to install it, but Steam Deck's KDE version can't read the theme file upside_down_face

Oh, that's weird. I'll give it a go with stock KDE. Maybe I forgot something in the theme file.

**Comment by Donnnno** [comment]
Thanks ! :)

**Comment by jtrees** [comment]
Turns out KDE Plasma can't handle the folder structure per se, and needs it all wrapped in a single archive instead. I added that to my fork:

[75c0318e54f882f63be1c84218132f068f969682](https://github.com/jtrees/Arcticons)

I also did a rebase so beware if you're pulling the changes.

I hope to add many more mappings (I mean currently there are like... 10 icons or so in the set) and improve the documentation (e.g. add a screenshot). Then I'll open a PR.

Let me know if this works now.
